### PR TITLE
Use LMAlertView as dynamic framework

### PR DESCRIPTION
**Pull request by BenchR267**
This fixes imports to be able to use LMAlertView as an dynamic framework + some code improvements for runtime safety.

### CODE REVIEW

File: LMAlertView/LMAlertView.h [hunk 1]
```diff
@@ -24,7 +24,7 @@
 - (float)velocity;
 @end
 #else
-#import <RBBSpringAnimation.h>
+#import "RBBSpringAnimation.h"
```
Comment (apwelsh) [inline LMAlertView/LMAlertView.h]: I believe the proper FrameWorks notation would be:
#import &lt;RBBAnimation/RBBSpringAnimation.h&gt;

File: LMAlertView/LMAlertView.m [hunk 2]
```diff
@@ -9,7 +9,7 @@
 #import "LMAlertView.h"
 #import "LMEmbeddedViewController.h"
 #import "LMModalItemTableViewCell.h"
-#import <CAAnimation+Blocks.h>
+#import "CAAnimation+Blocks.h"
```
Comment (apwelsh) [inline LMAlertView/LMAlertView.m]: Change to:
#import &lt;CAAnimationBlocks/CAAnimation+Blocks.h&gt;

**Review by apwelsh — CHANGES_REQUESTED** [review]
Suggested edit to LMAlertView.m

File: LMAlertView/LMAlertView.h (hunk 1 above)
Comment (BenchR267) [reply LMAlertView/LMAlertView.h]: It seems like I already updated this one

**Comment by BenchR267** [comment]
Any intentions in merging this?

**Comment by apwelsh** [comment]
I think the project was abandoned.  I ended up pulling in the source directly into my project and integrating the changes, only to drop its use completely for Apple's built in solution.

**Comment by ztalbot2000** [comment]
I still use it in my project.  It does what I need and still works in IOS
11.  I had pulled the source long time ago, and do not use it as a
framework.

Ttfn,
John Talbot

**Pull request closed without merging**